CMakeLists.txt

```diff
@@ -118,6 +138,7 @@ add_executable(fpm-accuracy
 )
 set_target_properties(fpm-accuracy PROPERTIES CXX_STANDARD 14)
 target_link_libraries(fpm-accuracy PRIVATE fpm libfixmath)
+endif()
```

Comment:
I think the `BUILD_ACCURACY` block should also contain the accuracy image generation.
And the `BUILD_BENCHMARK` block should also contain the performance data/image generation.

Reply:
I've improved PR according to the comment.